src/index.js

```diff
@@ -1,4 +1,17 @@
 import "./components";
+import { registerIconLibrary } from "@shoelace-style/shoelace";
+import { initHoneybadger } from "./_honeybadger";
 
 export * from "./controllers";
-export * from "./_honeybadger";
+export * from "./_honeybadger"; // Leaving this for legacy sake; should be removed once everyone is calling initialize below
+
+/** ***************************************** */
+/**  Apps should call this method on startup  */
+/** ***************************************** */
+export const initialize = () => {
```

Comment:
I'm just thinking about how this will look imported alongside lots of other libs in application.js -- we _can_ easily alias it, so it's not a big deal either way, but what do you think about `initTeamshares`, or a default export so we could do e.g. `import Teamshares from @shared-ui; Teamshares.start()`? Just musing about naming...